src/lib/tests/createMockNetworkLayer/index.ts

```diff
@@ -0,0 +1,26 @@
+import { IMocks } from "graphql-tools/dist/Interfaces"
+import getNetworkLayer from "relay-mock-network-layer"
+import { Network } from "relay-runtime"
+
+// Typescript thinks we're in React Native, this file should be test-only i.e. executed within Node.
+declare const __dirname: string
+// @ts-ignore
+import * as fs from "fs"
```

Comment:
Is there a better way to make TS happy here? `ts-jest` takes a [`tsConfig`](https://kulshekhar.github.io/ts-jest/user/config/tsConfig) option that looks promising, but I'm not knowledgeable enough about how the current setup works.

Reply:
Yeah, but that's worth doing in a separate PR IMO